src/components/UI/LanguageSelector/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import './index.css'
```

Comment:
The index.css does not contain any styling.

Reply:
Good catch!